source/crud/crud.rst

```diff
@@ -404,6 +404,19 @@ Read
      * NOTE: This option is deprecated in favor of timeoutMS.
      */
     maxTimeMS: Optional<Int64>;
+
+    /**
+     * Enables users to specify an arbitrary comment to help trace the operation through
+     * the database profiler, currentOp and logs. The default is to not send a value.
+     *
+     * The comment can be any valid BSON type for server versions 4.4.14 and above.
+     * For server versions between 4.4.0 and 4.4.14 string comment is supported.
+     # Servers versions below 4.4.0  do not support comment for count command,
+     * which is used to implement estimatedDocumentCount for server versions
+     * versions less than 4.9.0. Therefore, providing a comment may result
+     * in a server-side error.
```

Comment:
Per my comment below, let's add tests for the cases where we'd expect a server-side error:

 * Using a string comment on a pre-4.4 server version.
 * Using a document comment on a pre-4.4.14 sever version.